porcupine/plugins/anchors.py

```diff
@@ -0,0 +1,58 @@
+import tkinter
+import time
+from porcupine import tabs, get_tab_manager
+from porcupine.plugins import linenumbers
+
+# Dependent on code from linenumbers.py
+setup_after = ["linenumbers"]
+
+
+class AnchorManager:
+    def __init__(self, tab_textwidget: tkinter.Text, linenumber_instance: tkinter.Canvas) -> None:
+        self.tab_textwidget = tab_textwidget
+        self.linenumber_instance = linenumber_instance
+
+        self.anchor_symbol = "¶"
+
+        linenumber_instance.bind("<<Updated>>", self.update_linenumbers, add=True)
+
+    def toggle_on_off(self, event):
+        [cursor_index] = [self.tab_textwidget.index("insert linestart")]
```

Comment:
While `[foo] = [bar]` sets `foo` to `bar`, it is a rather complicated way to do so: it first creates a list containing `bar`, and then immediately unpacks the list, making `foo` be the only element.

```suggestion
        cursor_index = self.tab_textwidget.index("insert linestart")
```